pyintegration/README.md

```diff
@@ -1,20 +1,28 @@
 
-# Deephaven Engine Python Package
+# Deephaven Python Integration Package
 
-Deephaven Engine Python Package is created by Deephaven Data Labs. It allows Python application developers, data scientists
-to run Python scripts directly inside the Deephaven data servers to achieve maximum query performance in addition to taking advantage 
-of the unique capabilities of the Deephaven engine on ticking data and much more.
-
-## Status
-This package is in its very early development phase and subject to constant changes. THus until further notice, it is for internal use only.
+Deephaven Python Integration Package is created by Deephaven Data Labs. It allows Python developers including data
+scientists to access data, run queries, execute Python scripts directly inside the Deephaven data servers to achieve
+maximum performance. By taking advantage of the unique streaming table capability of Deephaven and its many data ingestion
+facilities (Kafka, Parquet, CSV, SQL, etc.), Python developers can quickly put together a realtime data process pipeline
+that is high performing and use to consume.
```

Comment:
This looks like the same block from sphinx, but it doesn't have the suggested changes.